cassandane/Cassandane/Cyrus/Caldav.pm

```diff
@@ -1320,6 +1320,79 @@ EOF
     );
 }
 
+sub test_invite_add_another_to_override
+    :VirtDomains :min_version_3_0 :needs_component_httpd
+{
+    my ($self) = @_;
+
+    my $service = $self->{instance}->get_service("http");
+    my $CalDAV = Net::CalDAVTalk->new(
+        user => "cassandane%example.com",
+        password => 'pass',
+        host => $service->host(),
+        port => $service->port(),
+        scheme => 'http',
+        url => '/',
+        expandurl => 1,
+    );
+
+    my $CalendarId = $CalDAV->NewCalendar({name => 'hello'});
+    $self->assert_not_null($CalendarId);
+
+    my $uuid = "6de280c9-edff-4019-8ebd-cfebc73f8201";
+    my $href = "$CalendarId/$uuid.ics";
+    my $card = <<EOF;
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:-//Apple Inc.//Mac OS X 10.10.4//EN
+CALSCALE:GREGORIAN
+BEGIN:VEVENT
+CREATED:20230806T234327Z
+UID:$uuid
+DTEND;TZID=Australia/Melbourne:20230831T183000
+TRANSP:OPAQUE
+SUMMARY:An Event
+DTSTART;TZID=Australia/Melbourne:20230831T153000
+DTSTAMP:20230806T234327Z
+SEQUENCE:0
+ATTENDEE;CN=Test User;PARTSTAT=ACCEPTED;RSVP=TRUE:MAILTO:cassandane\@example.com
+ATTENDEE;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:friend\@example.com
+ORGANIZER;CN=Test User:MAILTO:cassandane\@example.com
+RRULE:FREQ=WEEKLY
+END:VEVENT
+BEGIN:VEVENT
+CREATED:20230806T234327Z
+UID:$uuid
+RECURRENCE-ID;TZID=Australia/Melbourne:20230907T153000
+DTEND;TZID=Australia/Melbourne:20230907T183000
+TRANSP:OPAQUE
+SUMMARY:An Event
+DTSTART;TZID=Australia/Melbourne:20230907T153000
+DTSTAMP:20230806T234327Z
+SEQUENCE:0
+ATTENDEE;CN=Test User;PARTSTAT=ACCEPTED;RSVP=TRUE:MAILTO:cassandane\@example.com
+ATTENDEE;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:friend\@example.com
+ORGANIZER;CN=Test User:MAILTO:cassandane\@example.com
+END:VEVENT
+END:VCALENDAR
+EOF
+
+    $CalDAV->Request('PUT', $href, $card, 'Content-Type' => 'text/calendar');
+
+    $self->assert_caldav_notified(
+        { recipient => "friend\@example.com", is_update => JSON::false, method => 'REQUEST' },
+    );
+
+    $card =~ s/SEQUENCE:0/SEQUENCE:1/;
+    $card =~ s/RECURRENCE-ID/ATTENDEE;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:friend2\@example.com\nRECURRENCE-ID/;
+warn Dumper($card);
```

Comment:
Can you drop this warning please, or convert it an xlog if it's still useful? Thanks

Reply:
Good catch.  Removed